packages/mobile-top-nav/src/user-menu.js

```diff
@@ -5,7 +5,7 @@ class UserMenu extends TrackedElement {
   static get styles() {
     return css`
       :host {
-        --topOffset: -70vh;
+        --topOffset: -1300px;
```

Comment:
QUESTION: Will this also work when we go to 4k screens?

Reply:
Yes. Moving from relative to absolute units will ensure the element is in the same position no matter the viewport height. Since the user menu is shorter than this value, it will always be fully off screen.